agent/backend/pktvisor/pktvisor.go

```diff
@@ -471,6 +477,37 @@ func createExporter(ctx context.Context, logger *zap.Logger) (component.MetricsE
 	return exporter, nil
 }
 
+func createOtlpExporter(ctx context.Context, logger *zap.Logger) (component.MetricsExporter, error) {
+	// 2. Create the OTLP metrics exporter that'll receive and verify the metrics produced.
+	exporterCfg := &otlpexporter.Config{
+		ExporterSettings: config.NewExporterSettings(config.NewComponentID("otlp_exporter")), // Define the component id by config
+		TimeoutSettings:  exporterhelper.DefaultTimeoutSettings(),
+		QueueSettings:    exporterhelper.DefaultQueueSettings(),
+		RetrySettings:    exporterhelper.DefaultRetrySettings(),
+		GRPCClientSettings: configgrpc.GRPCClientSettings{
+			Endpoint:        "localhost:1234", // Define the export url by config
```

Comment:
Connecting this to orb-config is necessary even in this PR for it to be usable right?